README.md

```diff
@@ -94,7 +95,13 @@ We can search DebridMediaManager hash lists which are hosted on GitHub. This all
    ```
    GithubSettings__PAT=<YOUR TOKEN HERE>
    ```
+### Configure external access
 
+What you will need:
+1. Domain or subdomain that points toward your ip. You can use [DuckDNS](duckdns.org) for a free subdomain. [Installation instructions](http://www.duckdns.org/install.jsp) are provided to keep your IP updated.
+2. Ports 80 and 443 opened on your router/gateway adn forwarded to your KNightcrawler server. Refer to [PortForward.com](https://portforward.com/).
```

Comment:
Typos "adn" and "KNightcrawler"

Reply:
Thanks. I'll fix all of these and make a new PR.